source/developers/quickstarts/first_openedx_pr.rst

```diff
@@ -79,44 +83,21 @@ from the `official Docker documentation
 to the Docker daemon.
 
 
-Installing Tutor Nightly
+Installing Tutor Main
 ************************
 
-Navigate into a folder to hold your Open edX repositories.
-We will assume you're using ``~/openedx``, but you can choose any folder.
-
-.. code-block:: bash
+Tutor main is the best solution for using Tutor as your development
+environment and tracking the bleeding edge of the Open edX platform
+code.
 
-  mkdir -p ~/openedx
-  cd ~/openedx
-
-Create a Python 3.8 virtual environment (we'll call it ``tutor-venv``) and activate it:
-
-.. code-block:: bash
+Follow the official instructions for installing Tutor Main.
 
-  python3.8 -m venv tutor-venv
-  . tutor-venv/bin/activate
+.. admonition:: The Tutor documents will recommend this as well, but
```

Comment:
I don't know why this isn't working but I don't use this directive, I use `.. note::` or `.. warning::`

Reply:
@sarina sorted it out.  A combination of how the admonition title works and how the details are formatted.

Reply:
Actually, noticing that admonition provides 0 styling, which isn't super useful.  Does warning or another option provide styling?